packages/vendure-plugin-frequently-bought-together/README.md

```
@@ -0,0 +1,3 @@
+# Frequently Bought Together Plugin
+
+### [Official documentation here](https://pinelab-plugins.com/plugin/vendure-plugin-frequently-bought-together)
```

Markdownlint (0.35.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)